Test.Integration.Net7/Uitlities/EnvironmentVariable.cs

```diff
@@ -0,0 +1,252 @@
+﻿using Microsoft.Extensions.Configuration;
+using System.IO;
+
+namespace Test.Integration.Net7.Uitlities
+{
+    internal class EnvironmentVariable
+    {
+        private readonly IConfiguration configuration;
+
+        public EnvironmentVariable()
+        {
+            var builder = new ConfigurationBuilder()
+                 .SetBasePath(Directory.GetCurrentDirectory())
+                 .AddJsonFile("local.settings.json", optional: true, reloadOnChange: true);
+            configuration = builder.Build();
+        }
+
+        public string ApiKey
+        {
+            get
+            {
+                return configuration["Values:ApiKey"];
+            }
+        }
+        public string AccountId
+        {
+            get
+            {
+                return configuration["Values:AccountId"];
+            }
+        }
```

Comment:
only account_id and api_key should be taken from the env variables, the rest of the values should be directly in the test cases